docs/docs/user/references/cli-commands.md

```diff
@@ -43,6 +43,7 @@ $ kpops clean [OPTIONS] PIPELINE_PATH [COMPONENTS_MODULE]
 * `--defaults DIRECTORY`: Path to defaults folder  [env var: KPOPS_DEFAULT_PATH]
 * `--config FILE`: Path to the config.yaml file  [env var: KPOPS_CONFIG_PATH; default: config.yaml]
 * `--steps TEXT`: Comma separated list of steps to apply the command on  [env var: KPOPS_PIPELINE_STEPS]
+* `--filter-type [include|exclude]`: If the --steps option should include/exclude the steps  [default: FilterType.include]
```

Comment:
is there any way we can get this to show `default: include`? the avg user shouldn't be concerned with the fact that it's implemented as an enum behind the scenes